sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/testprofiles/NumericBooleansTestProfile.java

```diff
@@ -0,0 +1,21 @@
+package io.stargate.sgv2.docsapi.testprofiles;
+
+import com.google.common.collect.ImmutableMap;
+import io.quarkus.test.junit.QuarkusTestProfile;
+import java.util.Map;
+
+/**
+ * Simple test profile to have a switch to numeric booleans.
+ *
+ * <p>Annotate test class with @TestProfile(NumericBooleansTestProfile.class) to use.
+ */
+public class NumericBooleansTestProfile implements QuarkusTestProfile {
+
+  @Override
+  public Map<String, String> getConfigOverrides() {
+    // adapt consistency, depth and column name(s)
+    return ImmutableMap.<String, String>builder()
+        .put("stargate.data-store.secondary-indexes-enabled", "false")
```

Comment:
So this likely will be renamed or removed, based on our discussion this morning - right?

sai-enabled = true IS the same as "use numeric booleans" so I don't think this gives us anything anymore in the new simplification we discussed

Reply:
Yes whole profile is removed now.